python/ray/train/lightning/_lightning_utils.py

```diff
@@ -57,7 +62,7 @@ def get_worker_root_device():
 
 
 @PublicAPI(stability="beta")
-class RayDDPStrategy(DDPStrategy):
+class RayDDPStrategy(pl.strategies.DDPStrategy):
```

Comment:
Curious, would it make sense to import this after line 29?

Reply:
Although we can use `pl` as the prefix of the class path. We can not do `from pl.strategies import DDPStrategy`

Reply:
Oh why?

Reply:
hmm, this is the python syntax. We cannot import from an alias, but the full module path.